### PR TITLE
[Improvement] Add dynamic allocation patch for Spark 3.2

### PR DESCRIPTION
**Pull request by jerqi**
### What changes were proposed in this pull request?
Add the dynamic allocation patch for Spark 3.2, solve issue #106

### Why are the changes needed?
If we don't have this patch, users can't use dynamic allocation in Spark 3.2.

### Does this PR introduce _any_ user-facing change?
No

### How was this patch tested?
Manual test

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -154,7 +154,7 @@ rss-xxx.tgz will be generated for deployment
 ### Support Spark dynamic allocation
 
 To support spark dynamic allocation with Firestorm, spark code should be updated.
-There are 2 patches for spark-2.4.6 and spark-3.1.2 in spark-patches folder for reference.
+There are 3 patches for spark-2.4.6, spark-3.1.2 and spark-3.2.1 in spark-patches folder for reference.
```
Comment (frankliee) [inline README.md:157]: spark (2.4.6/3.1.2/3.2.1)

Comment (jerqi) [reply README.md:157]: why do we need this style?

Comment (frankliee) [reply README.md:157]: The original style can be too long, When are there more minor versions

Comment (jerqi) [reply README.md:157]: OK

**Review by colinmjj — APPROVED** [review]
LGTM

**Pull request merged**